packages/shared/.storybook/storybook.css

```
@@ -25,7 +25,7 @@
 
 /* Import admin-only M3 utilities used by Admin/* stories. */
 @import "../../admin/src/styles/admin-m3-tokens.css";
-@import "../../admin/src/styles/admin-m3-overrides.css";
+@import "../../admin/src/styles/admin-m3-components.css";
```

Stylelint (17.14.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)